### DISCUSSION TITLE
Use Web3Auth in Golang system

### DISCUSSION THREAD

**Discussion by TriNguyen102** [Web3Auth/Web3Auth#980] [Help]
Hello all. I'm implementing a back-end system and I want to use Web3Auth to create a credential connect Social Network with a Wallet. And my problem is I'm using Golang for my BE and in official document, you are just supporting for NodeJS.
So, did you open some API(s) or some End-point that I can send a request from Golang system to verify?
Thank you so much

**Comment by girishvi** [comment]
@yashovardhan

**Comment by agonist** [comment]
Hi I'm curious about that as well. thx !
